search/interfaces/graphql/schema.graphql

```diff
@@ -17,4 +17,25 @@ type Commerce_Search_Meta {
     page:           Int!
     numPages:       Int!
     numResults:     Int!
-}
\ No newline at end of file
+    sortOptions:    [Commerce_Search_SortOption!]
+}
+
+type Commerce_Search_SortOption {
+    label: String!
+    asc: String!
+    desc: String!
+    selectedAsc: Boolean!
+    selectedDesc: Boolean!
+}
+
+type Commerce_Search_Suggestion {
+    text:      String!
+    highlight: String!
+}
+
+type Commerce_Search_Result {
+    # Hits: []Document
```

Comment:
why not adding hits right now? Or if we dont need Commerce_Search_Result we can uncomment it completely?